.jenkins/library/vars/tests.groovy

```diff
@@ -504,12 +504,11 @@ def windowsLinuxElfBuild(String windows_label, String ubuntu_label, String compi
     }
 }
 
-def windowsCrossCompile(String label, String build_type, String lvi_mitigation = 'None', String OE_SIMULATION = "0", String lvi_mitigation_skip_tests = 'OFF', List extra_cmake_args = []) {
-    stage("Windows ${label} ${build_type} with SGX LVI_MITIGATION=${lvi_mitigation}") {
-        node(label) {
+def windowsCrossCompile(String label, String clang_version, String build_type, String lvi_mitigation = 'None', String OE_SIMULATION = "0", String lvi_mitigation_skip_tests = 'OFF', List extra_cmake_args = []) {
```

Comment:
To be either changed to clang_version or compiler across all tests so that it is consistent.